src/readbin/ida.ml

```diff
@@ -14,7 +16,10 @@ let run cmd =
   let r = In_channel.input_lines inp in
   In_channel.close inp; r
 
-let system cmd = Unix.system cmd |> ignore
+let system cmd =
+  let return = Sys.command cmd in
```

Comment:
yes, you're right, `get_symbols` can raise exceptions, and the rest of the code is written with the premise. So, your change is totally valid and correct. Thanks!